pyaerocom/io/read_mscw_ctm.py

```diff
@@ -324,6 +342,8 @@ def preprocess_units(units, prefix=None):
         new_unit = units
         if units == '' and prefix == 'AOD': #
             new_unit = '1'
+        elif units == '' and prefix == 'AbsCoef':
+            new_unit = '1/m'
         elif units == 'mgS/m2' or units == 'mgN/m2':
             raise NotImplementedError('Species specific units are not implemented.')
```

Comment:
I think this is "sort of" implemented now so it may be okay to remove this "elif" for N and S units.

Reply:
I tested it now, and I think the "sort of" implementation we did is not included in the release_0_10_0 branch.

Reply:
I added this back in for now.